Package.swift

```diff
@@ -35,6 +35,7 @@ let package = Package(
             dependencies: [
                 .product(name: "ServiceContextModule", package: "swift-service-context"),
                 .target(name: "Instrumentation"),
+                .target(name: "_CWASI", condition: .when(platforms: [.wasi])),
```

Comment:
We seem to be getting an error on this line on 5.10

> error: Target Tracing imports another target (_CWASI) in the package without declaring it a dependency.

Is this a bug in the package manager on 5.10? Can you take a look and if it is a bug we can just disable the strict dependency check on just 5.10

Reply:
@FranzBusch Indeed it does look like that is a bug. I tracked it down to `FB14859516` metnioned in https://forums.swift.org/t/do-swiftpm-conditional-target-dependencies-work/74047.

I removed the dependency check for swift versions before Swift 6. See https://github.com/apple/swift-distributed-tracing/pull/174/commits/386e324dbb9b6f3d3fe3eb0244ca6d96ba6fec63